### ISSUE TITLE
centerItem value being ignored on device

### ISSUE THREAD

**Issue by alpi** [kitasuke/PagingMenuController#39]
The centerItem value is being ignored (always true) on the device (Simulator works fine).

Add the following menuDisplayMode option to the Demo project to test it:
```
options.menuDisplayMode = PagingMenuOptions.MenuDisplayMode.FlexibleItemWidth(centerItem: false, scrollingMode: PagingMenuOptions.MenuScrollingMode.ScrollEnabledAndBouces)

**Comment by kitasuke** [comment]
Thanks for reporting. It reproduces here and I found related bugs as well...
I'm working on it!

**Comment by kitasuke** [comment]
I've just fixed the issue, so please use 0.4.4 version!
If it still occurs, please reopen this issue.
